src/components/ciConfig/CIConfig.scss

```diff
@@ -67,20 +83,95 @@
             }
         }
     }
+
     .popup-button {
         .select-button {
             height: 40px;
             padding-left: 10px;
         }
     }
+
     .target-platform__select__multi-value__label {
         font-size: 12px;
         padding: 2px;
         padding-left: 6px;
     }
 }
+
 .docker__dropdown {
     margin: auto;
     margin-right: 0;
     margin-top: 0;
 }
+
+.form__app-compose.config-override-view {
+    width: 100%;
+    margin: 0;
+    padding: 24px 0 16px;
+    margin-bottom: 0;
+}
+
+.form__app-compose {
+    &.config-override-view {
+        .white-card__docker-config {
+            .form-row__docker {
+                align-items: flex-end;
+            }
+        }
+
+        .allow-config-override {
+            &.cta {
+                top: 16px;
+                right: 16px;
+            }
+        }
+    }
+}
+
+.modal__body {
+    &.modal__config-override-diff {
+        width: 1024px;
+        height: 100%;
+        margin-top: 0;
+
+        .config-override-diff__view {
+            height: calc(100% - 40px);
+        }
+
+        .workflow {
+            &.workflow--create {
+                margin-bottom: 0;
+                border-bottom: none;
```

Comment:
can we use `dc__no-bottom-border` ?

Reply:
For using these classes I have to add additional checks so added the check at the parent and overridden the properties of the children wherever required in css.